crates/bevy_gizmos/src/primitives/helpers.rs

```diff
@@ -0,0 +1,94 @@
+use std::f32::consts::TAU;
+
+use bevy_math::{Mat2, Quat, Vec2, Vec3};
+use bevy_render::color::Color;
+
+use crate::prelude::{GizmoConfigGroup, Gizmos};
+
+// helpers - affine transform
+
+pub fn rotate_then_translate_2d(rotation: Mat2, position: Vec2) -> impl Fn(Vec2) -> Vec2 {
+    move |v| rotation * v + position
+}
+
+pub fn rotate_then_translate_3d(rotation: Quat, translation: Vec3) -> impl Fn(Vec3) -> Vec3 {
+    move |v| rotation * v + translation
+}
```

Comment:
These feel a bit unnecessary to me, but keeping them is probably fine

Reply:
Yeah, to be fair this seems to make only a minimal difference in terms of repetition. The idea here was to prevent the chance of making mistakes through typos, which would be higher if all the closures are inlined.